specification/components/schemas/cloudevents/nhsNumberChangeEventData.yaml

```
@@ -0,0 +1,49 @@
+type: "object"
+description: ""
+required:
+- nhsNumberStatus
+- fullUrl
+- provenance
+- versionId
+properties:
+  nhsNumberStatus:
+      type: "string"
+      description: |
+          A set of coded values that represent the status of the NHS which is the subject of the change. This field is only provided for the nhs-number-change-1 event type.
```

```suggestion
          A set of coded values that represent the status of the NHS Number which is the subject of the change. This field is only provided for the nhs-number-change-1 event type.
```

I think that's right?